matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/user/accountdata/AccountDataAPI.kt

```diff
@@ -36,4 +37,16 @@ internal interface AccountDataAPI {
             @Path("type") type: String,
             @Body params: Any
     )
+
+    /**
+     * Remove an account_data for the client.
```

Comment:
Maybe a bit better, but we may found better.

```suggestion
     * Remove an account_data for the user.
```

Reply:
I agree, done.